README.md

```diff
@@ -1 +1,50 @@
 # HangOps Code of Conduct
+
+To continue growing a safe, welcoming, and fair community, all HangOps members are empowered to call out and attempt to resolve disrespectful behavior, and admins are empowered to enforce the Code of Conduct.
```

Comment:
:thumbsup: There will be a list at the bottom of the CoC and we can find other ways to make this more readily available (perhaps an `@admins` Slack group).